src/views/TeamCreation.vue

```diff
@@ -52,15 +52,13 @@
             <b-form-input
             v-model="invites.memberNames[x - 1]"
             type="text"
-            required
```

Comment:
@liammoyn if we give an empty array for a List<Obj> in a request type, is that valid or 400?

Reply:
Should be valid, empty list for invites will just not send any emails